webapp/src/main/webapp/i18n/vivo_all.properties

```diff
@@ -1,812 +1,953 @@
-#
-# Text strings for the controllers and templates

```

Comment:
Unnecessary formatting changing make it challenging to determine if there are functional update buried in the noise. Please revert these updates.